sc-network/CMakeLists.txt

```diff
@@ -6,5 +6,5 @@ find_package(Qt5Core REQUIRED)
 find_package(Qt5Network REQUIRED)
 
 add_subdirectory(sctp_server)
-add_subdirectory(sctp_client)
+#add_subdirectory(sctp_client)
```

Comment:
Why this one is commented?

Reply:
Previously with `SC_BUILD_SCTP` option turned on both `sctp_server `& `sctp_client` build failed .
The pr fixes `sctp_server` as main part for sc-network. Build for `sctp_client` was commented here to pass build with `SC_BUILD_SCTP` option. `sctp_client` module should be fixed separately

Reply:
They should be fixed synchronously. We support whole code base. It is not allowed to comment some CMake branches they should be removed or supported

Reply:
Updated with fixes for sctp-client build